pycbc/workflow/coincidence.py

```diff
@@ -655,3 +688,31 @@ def setup_multiifo_interval_coinc(workflow, hdfbank, trig_files, stat_files,
 
     logging.info('...leaving coincidence ')
     return statmap_files
+
+def get_ordered_ifo_list(ifocomb, ifo_ids, insps):
+    # combination_prec stores precedence info for the detectors in the combination
+    combination_prec = {}
+    for ifo in ifocomb:
+        combination_prec[ifo] = ifo_ids[ifo]
+
+    # Use only the inspiral files required for the detector combination
```

Comment:
I don't see why this block is in this function, it doesn't seem to have much to do with ifo ordering or precedence. Can it be made a separate function, named eg `select_files_by_ifo_combination()` that just returns inspcomb?

Reply:
done